scripts/build_and_run_tests.bash

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+# Simple script to build swupd with multiple configurations and run all needed tests with different configurations
+
+set -e
+
+#configurable parameters
+JOB_COUNT=8
+CONFIGURE_ARGS="--with-fallback-capaths=$PWD/swupd_test_certificates --with-config-file-path=./testconfig --enable-third-party"
```

Comment:
`--enable-third-party` is the default now, so you can remove this.

Reply:
done